### PR TITLE
Set console mode for stdout and stderr in windows. Fixes #56

### LINKED ISSUE #56: `encore run` output control codes shown in Windows
After I start `encore run`, the formatting control codes are shown in Windows:

`←[90m8:35PM←[0m ←[32mINF←[0m registered endpoint ←[36mendpoint=←[0mWorld ←[36mpath=←[0m/hello.World ←[36mservice=←[0mhello`

### PR DESCRIPTION
**Pull request by frzam**
Please refer https://github.com/encoredev/encore/issues/56 

Git bash:
git-bash

Cmd:
cmd

Powershell:
ps

### CODE REVIEW

File: cli/cmd/encore/init_windows.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// terminal output.
+func init() {
+	setConsoleMode(windows.Stdout, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+	setConsoleMode(windows.Stderr, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+}
+
+// setConsoleMode enables VT processing on stout and stderr.
+func setConsoleMode(handle windows.Handle, flag uint32) {
+	var mode uint32
+	windows.GetConsoleMode(handle, &mode)
```
Comment (eandre) [inline cli/cmd/encore/init_windows.go:19]: If GetConsoleMode fails it'll leave mode as zero. If then SetConsoleMode succeeds we'll set the wrong mode. We should guard the SetConsoleMode behind an err check.

**Review by eandre — COMMENTED** [review]
Thanks for fixing this! Just a minor comment and then we should be good to go.

File: cli/cmd/encore/init_windows.go (hunk 1 above)
Comment (frzam) [reply cli/cmd/encore/init_windows.go]: Totally agree!

**Review by eandre — APPROVED** [review]
Looks great, thank you!

**Pull request merged**